GPU/Common/SoftwareTransformCommon.cpp

```diff
@@ -92,37 +92,34 @@ static void RotateUVThrough(TransformedVertex v[4]) {
 
 // Clears on the PSP are best done by drawing a series of vertical strips
 // in clear mode. This tries to detect that.
-static bool IsReallyAClear(const TransformedVertex *transformed, int numVerts) {
+static bool IsReallyAClear(const TransformedVertex *transformed, int numVerts, float x2, float y2) {
 	if (transformed[0].x != 0.0f || transformed[0].y != 0.0f)
 		return false;
 
 	// Color and Z are decided by the second vertex, so only need to check those for matching color.
 	u32 matchcolor = transformed[1].color0_32;
 	float matchz = transformed[1].z;
 
-	int bufW = gstate_c.curRTWidth;
-	int bufH = gstate_c.curRTHeight;
-
 	for (int i = 1; i < numVerts; i++) {
 		if ((i & 1) == 0) {
 			// Top left of a rectangle
-			if (transformed[i].y != 0)
+			if (transformed[i].y != 0.0f)
 				return false;
 			if (i > 0 && transformed[i].x != transformed[i - 1].x)
 				return false;
 		} else {
-			if ((i & 1) && (transformed[i].color0_32 != matchcolor || transformed[i].z != matchz))
+			if (transformed[i].color0_32 != matchcolor || transformed[i].z != matchz)
```

Comment:
```c++
if ((i & 1) == 0) {
    /// stuff
} else {
    if ((i & 1) != 0) {
        // other stuff
    }
}
```

It's not needed, we're already in the else condition for that check.

-[Unknown]